contents/handbook/engineering/feature-pricing.md

```diff
@@ -44,6 +44,12 @@ A good question to ask yourself here is, "If I were to switch away from PostHog,
 
 For example, if someone were to consider moving from PostHog to some other provider, cohorts would need to be manually recreated in the other provider, which would be tedious. However, something like Web Performance just happens and doesn't require any user involvement, so isn't sticky.
 
+### We should only charge users once per feature
+
+Our customers get confused when we charge for the same feature in multiple places. It's often hard to explain why we would do this.
+
+For example, when someone calls a feature flag, we send a $feature_flag_called event so we can have stats. In this case, we don't charge for those events, as our customers already pay for feature flags.
```

Comment:
Found this confusing as it didn't explain what it means to "charge for a feature in multiple places."

```suggestion
When one of our products has a dependency on another of our products, we should bundle the cost of the dependencies in with the product's pricing so customers only pay once for using a given product.

For example, when someone calls a feature flag, we send a $feature_flag_called event so we can have stats. In this case, we don't charge for those events, as the events are solely related to feature flags.
```